src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -617,6 +622,25 @@ class PickPage extends Component {
     });
   }
 
+  async clearPicklist() {
+    const { values } = this.state;
+    const { picklist } = values;
```

Comment:
Can't you just do:
```suggestion
    const { picklist } = this.state.values;
```
I do not see `values` in the code :thinking:

Reply:
yeah, you are right. Previously I used `stockMovementId` so I had to do it this way, and when changing it to `picklist`, I have not noticed I could shorten it. Thanks.